library/foundation/src/main/java/com/epmedu/animeal/foundation/topbar/TopBar.kt

```diff
@@ -32,7 +32,7 @@ fun TopBar(
         verticalAlignment = Alignment.CenterVertically,
     ) {
         if (navigationIcon == null) {
-            WidthSpacer(44.dp)
+            WidthSpacer(24.dp)
```

Comment:
Since we have different paddings on MoreScreen and FinishProfileScreen, let's add an argument for it

Reply:
I don't see the difference. Margin looks fine for me on both screens

Reply:
Updated

Reply:
@Fileek could you please check?